### ISSUE TITLE
[Bug]: VSCode PreCommit Hook Fails?

### ISSUE THREAD

**Issue by ll7** [una-auxme/paf23#59]
### Current Behavior

```txt
> git -c user.useConfigOnly=true commit --quiet --allow-empty-message --file -

Starting git hooks

No python files in commit, skip python linting
b5 1.3.3[0m
Found project path (/home/luttkule/git/paf23)[0m
Found Taskfile (Taskfile)[0m
Executing task python:lint[0m
[0m
[32mTask exited ok[0m
b5 1.3.3[0m
Found project path (/home/luttkule/git/paf23)[0m
Found Taskfile (Taskfile)[0m
Executing task markdown:lint[0m
[0m
[32mTask exited ok[0m
b5 1.3.3[0m
Found project path (/home/luttkule/git/paf23)[0m
Found Taskfile (Taskfile)[0m
Executing task lint[0m
[0m
[32mTask exited ok[0m

Finished git hooks

⧗    input: Add documentation for Discord webhook integration
⚠    scope may not be empty [scope-empty]
✖    subject may not be empty [subject-empty]
✖    type may not be empty [type-empty]

✖    found 2 problems, 1 warnings
ⓘ    Help: https://github.com/ll7/paf22/blob/main/doc/developement/commit.md

Error: '"$@"' exited with status 1
b5 exiting with status 1

[00;31mTask failed, see error above[0m

Error: '( export COMPOSE_PROJECT_NAME=paf23; export DOCKER_HOST_SYSTEM=linux; export DOCKER_HOST_USERNAME=luttkule; export DOCKER_HOST_UNIX_UID=1000; export DOCKER_HOST_UNIX_GID=1000; "$@" )' exited with status 1
b5 exiting with status 1

[00;31mTask failed, see error above[0m

Error: '( cd /home/luttkule/git/paf23/build && docker:run docker compose -f docker-compose.yml -f docker-compose.nvidia.yml "$@" )' exited with status 1
b5 exiting with status 1

[00;31mTask failed, see error above[0m

Error: '( cd /home/luttkule/git/paf23/build || return 1; if [ $command_strategy == 'exec' ]; then
    docker:compose exec "${options[@]}" "${exec_options[@]}" "$container" "$@";
else
    docker:compose run "${options[@]}" "${run_options[@]}" "$container" "$@";
fi )' exited with status 1
b5 exiting with status 1

[00;31mTask failed, see error above[0m
b5 1.3.3[0m
Found project path (/home/luttkule/git/paf23)[0m
Found Taskfile (Taskfile)[0m
Executing task comlipy[0m
[0m
[31mTask execution failed, see above[0m

```

### Expected Behavior

no error

### How to reproduce the issue

Any commit in VSCode?

**Comment by ll7** [comment]
```
git commit --no-verify -m "Your commit message"
```
circumvents the issue.

**Comment by MaxJa4** [comment]
Formatting the commit message in VSCode like this, "fixes" the issue (tested it yesterday): `fix(#99999): Commit message` or similar -> see [PAF22-Docs](https://github.com/ll7/paf22/blob/main/doc/02_development/03_commit.md)

**Comment by ll7** [comment]
The behavior is intentional and I should read the documentation more carefully.
